db/schema.rb

```diff
@@ -396,6 +396,19 @@
     t.index ["user_id"], name: "index_comments_on_user_id"
   end
 
+  create_table "consumer_apps", force: :cascade do |t|
+    t.boolean "active", default: true, null: false
+    t.string "app_bundle", null: false
+    t.string "auth_key"
+    t.datetime "created_at", precision: 6, null: false
+    t.string "last_error"
+    t.string "platform", null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["app_bundle", "platform"], name: "index_consumer_apps_on_app_bundle_and_platform", unique: true
+    t.index ["app_bundle"], name: "index_consumer_apps_on_app_bundle"
+    t.index ["platform"], name: "index_consumer_apps_on_platform"
```

Comment:
FYI for future reference: you don't really need an index on those separate coolumns if you have a composite unique index. That's because PostgreSQL is smart enough to recycle a composite index even when the `WHERE` condition in SQL only uses one of them.

Reply:
That's great to know, thanks for the tip! Since they're unnecessary I got rid of them

Reply:
Just to be clear, this works from left to right, i.e. if you have an index on a, b,c then Postgres can use it for queries on a, a and b, or a and b and c.